tests/unit/ingredients.t

```diff
@@ -562,6 +562,14 @@ Origin of peaches: Spain. Origin of some unknown ingredient: France. origin of A
 			ingredients_text => "each capsule contains: paracetamol 500 m 5 060198 790 0 mg.",
 		}
 	],
+	# Japanese additives
+	[
+		"ja-additives",
+		{
+			lc => "ja",
+			ingredients_text => "増粘剤(加工デンプン、キサンタン)、酢酸Na、トレハロース、加工 デンプン、グリシン、調味料(アミノ酸等)、酸化防 止剤(V.C, V.E)、着色料(野菜色素)",
```

Comment:
This is dubious because Japanese text should not contain spaces.

Reply:
ok, I found it in some products, I guess they were introduced by the OCR when there was a line feed. I'll remove the spaces.